tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "declarationMap": true,
+    "sourceMap": true,
+    "importHelpers": true,
+    "module": "commonjs",
+    "strict": true,
+    "target": "es2018",
+    "baseUrl": ".",
+    "outDir": "lib",
+    "rootDir": "src",
+    "resolveJsonModule": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "composite": true
+  },
+  "include": ["./src", "types/estlint-type-extension.d.ts"],
```

Comment:
Why do we need to include `types/estlint-type-extension.d.ts`?

Reply:
Now that I changed its type, we don't need this anymore. Removed it.